src/modal/src/components/__snapshots__/Modal.test.js.snap

```diff
@@ -268,220 +268,521 @@ exports[`modal snapshots should match snapshot when isVisible="true" 1`] = `
     shouldCloseOnClick={true}
   >
     <Portal>
-      <StyledOverlayComponents__OverlayOuterContainer>
-        <div
-          className="c0"
-        >
-          <StyledOverlayComponents__OverlayInnerContainer
-            onClick={[Function]}
+      <Portal
```

Comment:
Please revert. I'd prefer to not update the Modal snapshot, as I'm still looking into the changes that happened in the various dependencies to understand why this has changed, and to figure out how to move forward.

Reply:
I will revert the snapshots as requested. 
As I understand it, this will cause the test suite to fail.

Reply:
Correct.